sea-orm-cli/src/cli.rs

```
@@ -417,23 +428,17 @@ pub async fn main() {
     let verbose = cli.verbose;
 
     match cli.command {
+        Commands::Config { command } => run_config_command(command).unwrap_or_else(handle_error),
         Commands::Generate { command } => {
             run_generate_command(command, verbose)
                 .await
                 .unwrap_or_else(handle_error);
         }
         Commands::Migrate {
-            migration_dir,
+            migration_dir: _,
```

CLI flags should override the configuration file, not be silently ignored.